core/core.go

```diff
@@ -56,6 +58,25 @@ func createInitialOfflineState() error {
 	return nil
 }
 
+func startCleanupTimer() {
+	_cleanupTicker := time.NewTicker(5 * time.Minute)
+	go func() {
+		for {
+			select {
+			case <-_cleanupTicker.C:
+				resetDirectories()
```

Comment:
@gabek something I just realized, this happens no matter whether the stream is still connected or not. 😬 unless I am missing something

Reply:
I guess it doesn't help that we just tried to back port a change by copying/pasting from another branch.  If this can wait until 0.0.3 it's working over there.

In the 0.0.3 HLS rework branch here's the actual change:

https://github.com/owncast/owncast/commit/151316131af6526417cbeabaf13c0eec6a903c01#diff-f75ca533d67f6226e2d6a028714c8c92

Reply:
But yeah, it only needs to run once, so it can be stopped after it runs after that first 5min delay.  But I'm not sure why you're seeing it continue to run even after a stream starts.  Seems like the ticker never is told to stop?

Reply:
I've been testing with the HLS branch, and on goth.land every time I reconnect it's due to restarting the server, so I haven't seen this, but it's totally resetting after a second RTMP connection.

If you run with `--enableVerboseLogging` it'll log out when it runs that reset.

![image](https://user-images.githubusercontent.com/414923/94099575-209ec880-fde0-11ea-8468-71d62b07d115.png)

Reply:
omg!

`_cleanupTicker := time.NewTicker(5 * time.Minute)`
should have been
`_cleanupTicker = time.NewTicker(5 * time.Minute)`

Reply:
Hopefully this fixes it.  Testing more... https://github.com/owncast/owncast/commit/cb2794f68c029cd205addd9d5a97fe5b16c92d10

Reply:
Oh my 🙈  I didn't even realize that! Whoopies! I'll test Sunday :)